.github/workflows/README.md

```diff
@@ -0,0 +1,126 @@
+# AlphaX Release Process
+
+## Release a new version
+### Tagging of changes
+In order to have release notes automatically generated, changes need to be tagged with labels.
+The following labels are used (should be safe-explanatory):
+`breaking-change`, `bug`, `enhancement`.
+
+### Release a new version
+Note: Releases need to be done from the `main` branch.
+
+1. Bump the version locally to (e.g. to `X.Y.Z`) and merge the change to `main`.
+2. Create a new draft release on GitHub using the
+[Create Draft Release](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml) workflow.
```

Comment:
manually executing the GitHub action?

Reply:
Yes, at the moment. We could also change this in the future such that you just add a tag (e.g. `vX.Y.Z`) to a commit and then the action starts immediately (learnt that this works only after implementing it differently ;-))